src/functions/helper_lookup.ts

```diff
@@ -37,7 +37,8 @@ export function assertDomainLength(domain: Maybe<FunctionResultObject>[]) {
 
 export function addPivotDependencies(
   evalContext: EvalContext,
-  coreDefinition: PivotCoreDefinition
+  coreDefinition: PivotCoreDefinition,
+  forMeasures: PivotCoreMeasure[]
 ) {
   //TODO This function can be very costly when used with PIVOT.VALUE and PIVOT.HEADER
```

Comment:
unrelated: what's the state of this TODO ?

Reply:
Think it's still a todo until we find it to be a real bottleneck ;-)